modules/core/src/viewports/web-mercator-viewport.js

```diff
@@ -109,7 +112,7 @@ export default class WebMercatorViewport extends Viewport {
 
       // projection matrix parameters
       orthographic,
-      fovyRadians: fov,
+      fovyRadians,
       aspect,
       focalDistance,
```

Comment:
I think `focalDistance` always needs to be set to `altitude`. You can test its effect with sizeUnits/radiusUnits/widthUnits: 'pixels'.

Reply:
Even if I change that the size of the points is incorrect (they are too large by a factor of about 2.25) when I test it against the Google projection. `2.2553542518310286` I recognize as `fovyToAltitude(fovy)` for that projection (while `altitude` is `1`, so my feeling is that there is somewhere in deck.gl that making the assumption that `fovyToAltitude(fovy) === altitude` but I haven't yet found where

Reply:
I've set focalDistance to always equal altitude, as (as per offline discussion) it turns out we can make this work with the Google matrices if we adjust the near plane